framework/src/ble_manager/ble_manager_lwnl.c

```diff
@@ -259,6 +259,28 @@ trble_result_e ble_drv_operation_enable_notification(trble_operation_handle *han
 	return res;
 }
 
+trble_result_e ble_drv_operation_enable_indication(trble_operation_handle *handle)
+{
+	trble_result_e res = TRBLE_SUCCESS;
+	lwnl_msg msg = {BLE_INTF_NAME, {LWNL_REQ_BLE_OP_ENABLE_INDICATE}, sizeof(trble_operation_handle *), (void *)handle, (void *)&res};
+	if (_send_msg(&msg) < 0) {
+	dbg("res %d\n", res);
```

Comment:
Please add one more tab for alignment.

Reply:
fix done, please check via https://github.com/Samsung/TizenRT/pull/5637/commits/fcafbac57359fd9cdabc0dd9a1f2e0d97c931cae